daml-lf/kv-support/src/main/scala/com/daml/lf/kv/transactions/TransactionConversions.scala

```diff
@@ -92,9 +95,157 @@ object TransactionConversions {
       .sequence_
       .map(_ => RawTransaction(transactionBuilder.build.toByteString))
   }
+
+  def decodeContractIdsAndKeys(
```

Comment:
The name here isn’t all that helpful. I’d suggest renaming it to something that contains `outputs` and adding a doc comment.

Reply:
Agreed, thanks. Changed.